.github/actions/build_envoy/action.yaml

```diff
@@ -66,7 +66,7 @@ runs:
 
         # Compile Envoy
         # https://github.com/envoyproxy/envoy/blob/main/bazel/README.md#release-builds
-        bazel --bazelrc=/dev/null build --jobs=auto -c opt //source/exe:envoy-static.stripped
+        bazel --bazelrc=/dev/null build --jobs=auto -c opt //source/exe:envoy-static.stripped --config=libc++
```

Comment:
I'm not confident that all the necessary dynamic libraries are installed on the base image (`ghcr.io/cybozu/ubuntu:22.04`).
Could you please add a post-build step in `build_envoy` to check if envoy binary can start correctly?

For example, we can test it like:
```
$ docker run --rm -it ghcr.io/cybozu/envoy:<ENVOY_VERSION> envoy --version

envoy  version: 0de8b2b94c75dbe8c2f897058e16d23d959783fa/1.28.1/Modified/RELEASE/BoringSSL
```